packages/kirakira-backend/DefaultApi.md

```diff
@@ -340,6 +342,60 @@ No authorization required
 
 [[Back to top]](#) [[Back to API list]](README.md#documentation-for-api-endpoints) [[Back to Model list]](README.md#documentation-for-models) [[Back to README]](README.md)
 
+# **recommendations**
+> Array<Videos200ResponseVideosInner> recommendations()
+
+
+### Example
+
+
+```typescript
+import {  } from '';
+import * as fs from 'fs';
+
+const configuration = .createConfiguration();
```

Comment:
seems  you need to specify the export and package name somewhere in the openapi config

Reply:
alright let's try it...

Reply:
nvm I don't see what the issue is, I'm already specifying npmName. To what extent does it matter?

Reply:
not much, just makes the docs look ugly and not copy-pastable